core/src/main/scala/sbt/librarymanagement/Zero.scala

```diff
@@ -0,0 +1,13 @@
+package sbt.librarymanagement
+
+trait Zero[A] {
+  def zero: A
+}
+
+object Zero {
+  def apply[A](z: => A): Zero[A] = new Zero[A] {
+    def zero: A = z
+  }
+  implicit val StringZero: Zero[String] = Zero[String]("")
+  implicit val LongZero: Zero[Long] = Zero[Long](0L)
```

Comment:
I don't think we should introduce such a typeclass universally in sbt.librarymanagement.

Let's move it into VersionNumber, so it's a VersionNumber specific utility only.

Reply:
done b67a16e